src/config.rs

```diff
@@ -669,13 +679,25 @@ pub struct PrimaryColors {
     foreground: Rgb,
 }
 
+#[derive(Debug, Deserialize)]
+pub struct CursorColors {
```

Comment:
The `PrimaryColors` type seems equivalent to this except in semantics. Maybe `PrimaryColors` could be renamed to something more general. In any case, we probably don't need two identical structs with different names.